tests/integration_python/pixi_global/test_global.py

```diff
@@ -1890,3 +1890,24 @@ def test_remove_dependency(pixi: Path, tmp_pixi_workspace: Path, dummy_channel_1
         env=env,
         stderr_contains="Environment dummy-a doesn't exist",
     )
+
+
+def test_menuinst_install(pixi: Path, tmp_pixi_workspace: Path, dummy_channel_1: str) -> None:
+    env = {"PIXI_HOME": str(tmp_pixi_workspace)}
```

Comment:
Would be good to avoid cluttering the machine with things installed by our tests.
For Linux setting `HOME` should do. Let's check what's the best approach for macOS and Windows

Reply:
I'm not sure what you mean to say with this?

Reply:
We solved the confusion